### PR TITLE
Add handling for view features button

### PR DESCRIPTION
**Pull request by FuadNabi**
I added handling for view features button

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+  } else {
+    downloadBtn.style.backgroundColor = 'green';
+    toggle = "deqiq-olsun"
+  }
+}
 
+viewFeaturesdBtn.onclick = () => {
   if ( toggle == "deqiq-olsun" ) {
-    downloadBtm.style.backgroundColor = 'red';
+    viewFeaturesdBtn.style.backgroundColor = 'blue';
```
Comment (8Bts) [inline script.js]: Lets use `orange` here isntead of `blue`

**Review by 8Bts — CHANGES_REQUESTED** [review]
Check the comment above

**Comment by FuadNabi** [comment]
@8Bts  I made the requested changes

**Review by 8Bts — APPROVED** [review]
LGTM

**Pull request merged**